src/ext/framework/help_commands.rs

```diff
@@ -0,0 +1,183 @@
+pub use ext::framework::command::{Command, CommandType, CommandGroup};
```

Comment:
This can be simplified to `super::{Command, CommandGroup, CommandType}`. If they're being re-exported, that should be done from `ext/framework/mod.rs`